deployment/v3/docs/external services configuration.md

```diff
@@ -0,0 +1,82 @@
+#  External Services Configuration
+
+This document describes how to configure external services like MinIO, Keycloak, and Postgres.
+
+
+## Postgres external configuration procedure
+
+* Go to `mosip-config` and checkout to a specific branch to update the `config` properties.
+* The default value for the database hostname is `postgres-postgresql.postgres` and the default value for the database port number are `5432`.
+* If you are using an external service, then you have to update the hostname and port number with the `sed` command mentioned below and also provide the external-hostname along with the external-port. 
+  ```
+   cd mosip-config
+   sed -i 's/postgres-postgresql.postgres/<postgres-external-hostname>/g' *
+   sed -i 's/5432/<postgres-external-port>/g' *
+  ```
+* Create a `postgres` namespace.
+  ```
+   kubectl create ns postgres
+  ```
+* Create a secret with the name `postgres-postgresql` in the `postgres` namespace, with the appropriate superuser password.
+* For example:
+  ```
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: postgres-postgresql
+    namespace: postgres
+  type: Opaque
+  data:
+    postgresql-password: {{ base64 encoded superuser password }}
+  ```
+* Proceed with **postgres-init** script from [here](../external/postgres/README.md#initialize-db).
+
+
+## Keycloak external configuration procedure
+
+* Create a `keycloak` namespace.
+  ```
+   kubectl create ns keycloak
+  ```
+* Create configmap with the name `keycloak-host` in the `keycloak` namespace and with the following data. Use appropriate hostname and URLs.
+* For example:
+  ```
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: keycloak-host
+    namespace: keycloak
+  data:
+    keycloak-external-host: iam.sandbox.mosip.net
+    keycloak-external-url: https://iam.sandbox.mosip.net
+    keycloak-internal-host: keycloak.keycloak
+    keycloak-internal-url: http://keycloak.keycloak
+  ```
+* Create a secret with the name `keycloak` in the `keycloak`namespace with the following data.
```

Comment:
Create a secret with the name `keycloak` in the `keycloak` namespace with the following data. (space missing after the wors 'keycloak').